code/rounds/InProgressRound.cs

```diff
@@ -0,0 +1,112 @@
+using Sandbox;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TTTReborn.Player;
+
+namespace TTTReborn.Rounds
+{
+    public class InProgressRound : BaseRound
+    {
+        public override string RoundName => "In Progress";
+        public override int RoundDuration => TTTReborn.Gamemode.Game.TTTRoundTime;
+        public override bool CanPlayerSuicide => true;
+        
+        public List<TTTPlayer> Spectators = new();
+        
+        private bool _isGameOver;
+
+        public override void OnPlayerKilled( TTTPlayer player )
+        {
+            Players.Remove( player );
+            Spectators.Add( player );
+            
+            player.MakeSpectator( player.EyePos );
+
+            if ( IsRoundOver() )
+            {
+                _ = ChangeToPostRound();
+            }
+        }
+        
+        public override void OnPlayerLeave( TTTPlayer player )
+        {
+            base.OnPlayerLeave( player );
+
+            Spectators.Remove( player );
+
+            if ( IsRoundOver() )
+            {
+                _ = ChangeToPostRound();
```

Comment:
I've seen this quite a few times and I don't understand the purpose of `_`. Could you explain this?

Reply:
It tells the compiler that you don't plan on using the return value for anything. Used in the FP code as well, so I'm assuming it is useful to do.